python/tvm/relax/transform/transform.py

```diff
@@ -475,18 +475,20 @@ def multiply(
 
 
 def MetaScheduleApplyDatabase(
-    work_dir: Optional[str] = None,
+    work_dir: Optional[str] = None, default_schedule_fallback=False
 ) -> tvm.ir.transform.Pass:
     """Apply the best schedule from tuning database.
     work_dir : Optional[str]
        work directory to deduce default database if database is not provided
        (it will be ignored when an user passes database)
+    default_schedule_fallback : bool
+       whether to fallback to default schedule when no schedule is found in database
     Returns
     -------
     ret : tvm.transform.Pass
         The registered pass
     """
-    return _ffi_api.MetaScheduleApplyDatabase(work_dir)  # type: ignore
+    return _ffi_api.MetaScheduleApplyDatabase(work_dir, default_schedule_fallback)  # type: ignore
```

Comment:
Let us instead do a compositional pass, so in this case.  We can instead have a flag `warn_missing_entry`.
Which can be set to false.

Then we can compose the two pass together without having to injecting fallback logic into MSApplyDB